client/my-sites/backup/backup-contents-page/index.tsx

```diff
@@ -58,7 +58,32 @@ const BackupContentsPage: FunctionComponent< OwnProps > = ( { rewindId, siteId }
 					<div className="backup-contents-page__header daily-backup-status status-card">
 						<div className="status-card__message-head">
 							<img src={ cloudIcon } alt="" role="presentation" />
-							<div className="status-card__title">{ translate( 'Backup contents from:' ) }</div>
+							<div className="status-card__header-content">
+								<div className="status-card__title">{ translate( 'Backup contents from:' ) }</div>
+								<div className="status-card__learn-about">
+									<a
+										href="https://jetpack.com/blog/introducing-backup-file-browser/"
+										target="_blank"
+										rel="noreferrer"
+									>
+										{ translate( 'Learn about file browser' ) }
```

Comment:
I think we should probably use `Learn about the file browser` here.

Reply:
Totally agreed. I've updated the PR.